### PR TITLE
feat: two variants of certs

### PR DESCRIPTION
**Pull request by uzairr**
## Description:
This PR will add executive education variants for certificate type display.

For better understanding of executive education please visit this [Confluence page](https://openedx.atlassian.net/wiki/spaces/IM/pages/3333161057/Executive+Education+Course+Type)

## Related PRs:
1: [edx-platform](https://github.com/openedx/edx-platform/pull/29999)
2: [course-discovery](https://github.com/openedx/course-discovery/pull/3310)
## Linked Ticket:
for more details please visit: [PROD-2708](https://openedx.atlassian.net/browse/PROD-2708)

### CODE REVIEW

File: ecommerce/courses/utils.py [hunk 1]
```diff
@@ -133,7 +133,9 @@ def get_certificate_type_display_value(certificate_type):
         'honor': _('Honor'),
         'professional': _('Professional'),
         'verified': _('Verified'),
-        'executive-education': _('Executive Education')
+        'executive-education': _('Executive Education'),
+        'paid-executive-education': _('Paid Executive Education'),
+        'unpaid-executive-education': _('Unpaid Executive Course'),
```
Comment (DawoudSheraz) [inline ecommerce/courses/utils.py]: "Unpaid Executive Education" as opposed to unpaid executive course

**Review by azanbinzahid — APPROVED** [review]

**Review by DawoudSheraz — APPROVED** [review]
To add Enrollment track document and related PRs in platform and discovery in PR description.

**Review by inventhouse — APPROVED** [review]
Looks reasonable

**Comment by dianekaplan** [comment]
@DawoudSheraz FYI we noticed that this was sitting in the pipeline un-deployed, and we needed a clean pipeline for another change to go through today so we're reverting it [here](https://github.com/openedx/ecommerce/pull/3679). When you do add back your change, please do so at a time when you're able to verify and deploy as usual, [as per our policy](https://openedx.atlassian.net/wiki/spaces/RS/pages/1835106870/How+to+contribute+to+our+repositories). Let us know if you have any questions!

**Comment by DawoudSheraz** [comment]
Thank you, @dianekaplan. 
@uzairr to follow-up here.

**Comment by DawoudSheraz** [comment]
Re-adding the changes in https://github.com/openedx/ecommerce/pull/3682

**Pull request merged**